packages/ui-components/style/icons/clear-pipeline.svg

```diff
@@ -1,4 +1,4 @@
-<svg focusable="false" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" fill="currentColor" width="16" height="16" viewBox="0 0 32 32" aria-hidden="true">
+<svg id="icon" focusable="false" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" fill="currentColor" width="16" height="16" viewBox="0 0 32 32" aria-hidden="true">
```

Comment:
svg files shouldn't have id's in them like this since multiple instances of it can be added to the dom, this is what was causing the R icons issues we were dealing with last sprint. if this was added for css reasons instead use class